.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,6 +9,7 @@ assignees: ''
 
 <!--- Provide a general summary of the Issue in the Title above -->
 
+
```

Comment:
Can you maybe drop commit 09fa1c2a4f40873ee5d7aef3d2b92a26750aae9a which contains this change? It is also not signed...